src/storage/src/table/mem_table.rs

```diff
@@ -45,26 +46,71 @@ impl MemTable {
     }
 
     /// read methods
-    pub async fn get_row(&self, pk: &Row) -> StorageResult<Option<RowOp>> {
-        todo!()
+    pub fn get_row(&self, pk: &Row) -> StorageResult<Option<&RowOp>> {
+        Ok(self.buffer.get(pk))
     }
 
     /// write methods
-    pub async fn insert(&mut self, _pk: Row, _value: Row) -> StorageResult<()> {
+    pub fn insert(&mut self, pk: Row, value: Row) -> StorageResult<()> {
+        let entry = self.buffer.entry(pk);
+        match entry {
```

Comment:
I believe this part can be better rewritten using `entry`'s `or_insert_with` and `and_modify` API. Can be done in later PRs.